stats/Exemplars.md

```diff
@@ -0,0 +1,46 @@
+Exemplars
+===
+
+Exemplars are example data points for aggregated data. In particular, for distribution-type
+metrics, exemplars are points associated with each bucket in the distribution giving an
+example of what was aggregated into the bucket.
+
+Exemplars consist of a value that was recorded, an exact timestamp of when that value
+was recorded, and a set of attachments.
+
+Attachments
+---
+
+Exemplars my have zero or more attachments. Attachments are key-value pairs that
+describe the context in which the exemplar was recored.
+
+Well-known attachment keys
+---
+
+We define the following types of attachments:
+
+1. `TraceID`: hex encoded trace ID
```

Comment:
I would say lower base16 because we say that the trace ID is byte[16] not a number. Also that ensures this has 32 hex characters all the time.